### ISSUE TITLE
Level 24 too abrupt

### ISSUE THREAD

**Issue by solsend2l** [thomaspark/flexboxfroggy#21]
This is a really awesome tutorial, good job, thanks so much!

The only suggestion is warming up to level 24 combined concepts, especially wrap-reverse. That's pretty intense!

**Comment by thomaspark** [comment]
Hi @solsend2l, thanks for your comments!

I agree that level 24 is pretty tough, but I intended it to be like that to give the opportunity to play around with all of the properties one last time, and as a "boss level" final challenge to add a sense of accomplishment.

I hope you found the previous 23 levels to have a smooth learning curve.

**Comment by SergiOnGit** [comment]
what is right code? :D

**Comment by NdYAG** [comment]
Figure out that `transform: scale(-1, 1)` works as a replacement for `flex-wrap: wrap-reverse`.
Only for this example as it's symmetrical.

**Comment by solsend2l** [comment]
Honestly I had to cheat, look at [code](https://github.com/thomaspark/flexboxfroggy/blob/gh-pages/js/levels.js#L343) :tongue:

**Comment by AndyHoang** [comment]
Solve them all! 
Very nice play with the frogs.
These ex will be played plenty of times

**Comment by stuartpb** [comment]
Yeah, considering that no level prior to level 24 has you use `wrap-reverse` (it's just mentioned without having the user try it), I think it's a little too abrupt. All that's needed is an additional level after the introduction of `flex-wrap` that demonstrates `wrap-reverse` interactively.

**Comment by stuartpb** [comment]
Also, it wouldn't hurt to have a little buildup combining a few of the concepts before the final one - including one or two that combine `order` with things like `flex-wrap`.

**Comment by gabrielkunkel** [comment]
+1 Agreed. I think this would be be better with some more exercises. My brain went *Woh!* when I got to 24. I'm glad to find part of the answer here. 

This is a fantastic tutorial, that still has so much more potential.

**Comment by jameskopacz** [comment]
Yeah, I didn't even know ```wrap-reverse``` existed until I gave up and came here via the github link on the site.

I didn't think want to google for an answer since I thought I was taught everything I needed to know to complete the problem :(


@SergiOnGit Here's what I ended up with after learning about ```wrap-reverse```:
```css
flex-flow: column-reverse wrap-reverse;
justify-content: center;
align-content: space-between;
```

**Comment by jsnmoon** [comment]
I think having a wrap-reverse exercise somewhere before level 24 would ease the learning curve here.

**Comment by based2** [comment]
24 seems ok, for me.

**Comment by MathRobin** [comment]
@ghost thanks for the solution. I was not able to find it by myself.

Not sure to understand why is it the solution... Maybe provide solution with explanations could be useful in the game

**Comment by Esslamb** [comment]
I agree 24 hit as a bit of a shock but it forced me to find the answer elsewhere and in the process found other resources.

**Comment by krystian3w** [comment]
```
flex-flow: column wrap;
justify-content: center;
align-content: space-between;
transform: scale(-1, -1);
```

Upside down 🐸  working too :stuck_out_tongue_winking_eye:

**Comment by Albert-Gao** [comment]
Besides the `wrap-reverse`, the true value of level 24 is I now know that with one class, you can do so much..... Otherwise, I will apply other class for the children.haha. Really thanks.

**Comment by aidalgol** [comment]
> I agree that level 24 is pretty tough, but I intended it to be like that to give the opportunity to play around with all of the properties one last time, and as a "boss level" final challenge to add a sense of accomplishment.

I like the concept of having a boss level, but I think there need to be just a few free-form exercises leading up to it.

**Comment by rainstormza** [comment]
`flex-direction: column-reverse;
flex-wrap: wrap-reverse;
justify-content: center;
align-content: space-between;`

**Comment by rainstormza** [comment]
flex-direction: column-reverse;
flex-wrap: wrap-reverse;
justify-content: center;
align-content: space-between;

**Comment by ys3322** [comment]
`flex-flow: column-reverse wrap-reverse;
align-content: space-between;
justify-content: center;`
